src/components/posts/PostsPage.js

```diff
@@ -1,71 +1,154 @@
-    event.preventDefault();
+    event.preventDefault()
     client
       .post('/post', post)
       .then((res) => {
-        setPostResponse(res.data);
-        setPosts((posts) => [res.data.data.post, ...posts]);
+        setPostResponse(res.data)
+        setPosts((posts) => [res.data.data.post, ...posts])
+        setLoad((x) => !x)
+      })
+      .catch((data) => {
+        console.log(data)
       })
-      .catch((data) => {});
-    setPost(() => ({ content: '' }));
-  };
+    setPost(() => ({ content: '' }))
+  }
 
   const handleChange = (event) => {
-    event.preventDefault();
-    const { value, name } = event.target;
+    event.preventDefault()
+    const { value, name } = event.target
     setPost({
       ...post,
       [name]: value,
-    });
-  };
+    })
+  }
+
+  const createComment = async (event, postId) => {
+    event.preventDefault()
+    client
+      .post(`/post/${postId}/comment`, { comment })
+      .then(() => {
+        setLoad((x) => !x)
+        setComment('')
+      })
+      .catch((data) => {
+        console.log(data)
+      })
+  }
+
+  const addToShowedComments = (id) => {
+    !showAllArr.some((a) => a === id) && setShowAll((x) => [...x, id])
+  }
+
+  const removeFromShowedComments = (id) => {
+    setShowAll((x) => x.filter((c) => c !== id))
+  }
+
+  const handleComment = (event) => {
+    event.preventDefault()
+    const { value } = event.target
+    setComment(value)
+  }
 
   return (
     <>
       <Header role={role} />
       <section className='posts-section'>
         {postResponse.status}
-        <PostForm handleSubmit={createPost} handleChange={handleChange} inputValue={post.content} />
-        <ul className="posts-list">
+        <PostForm
+          handleSubmit={createPost}
+          handleChange={handleChange}
+          inputValue={post.content}
+        />
+        <ul className='posts-list'>
           {posts.map((post, index) => (
             <li key={index} className='post-item'>
               <Box>
-                <div className="post-content">{post.content}</div>
-                <Stack className="names-date" spacing={2} direction="row">
+                <div className='post-content'>{post.content}</div>
+                <Stack className='names-date' spacing={2} direction='row'>
                   <Link to={`/user/${post.user.id}`} className='post-author'>
-                    <Box className="fullname" variant='contained'>
+                    <Box className='fullname' variant='contained'>
                       <strong>{`${post.user.profile.firstName} ${post.user.profile.lastName}`}</strong>
                     </Box>
                   </Link>
-                  <Box className="date-time" variant="contained">
+                  <Box className='date-time' variant='contained'>
                     {dateTimetoRelativeTime(post.createdAt)}
                   </Box>
                 </Stack>
               </Box>
+              <div className='comments-section'>
```

Comment:
![image](https://user-images.githubusercontent.com/15673068/166872925-8507c3f9-5e0f-45e0-9c55-10ff1dd99cd6.png)

When I type in a comment, the comment text is displayed in every comment box on the page - it looks a bit odd! Can this be changed?

Reply:
Sure! but this is another task (another card will be handled by Barry) and we wanna do it in another branch (answering the first comment)

I will fix this bug after we refactor this code and create a separate component.

Reply:
Hmmm...I'm not sure that's the best approach as it means there are bugs in what you have here. Since we're working with the idea of  a "stable main" anything you merge should be working, if not compelete. So you don't need to do all the features, but the features you do complete should be finished (bug free, styling complete, refactored). 

Sorry but you'll need to fix this one!

Reply:
will do